test/run_test.sh

```diff
@@ -64,6 +64,12 @@ $PYCMD test_cuda.py $@
 echo "Running NCCL tests"
 $PYCMD test_nccl.py $@
 
+echo "Running C++ Extensions tests"
+cd cpp_extensions
+$PYCMD setup.py install
```

Comment:
This is going to install to the user's actual site-packages, which means that it's not concurrent safe and an existing install could goo up the works. Unfortunately I can't think of a convenient way to created a stacked virtual environment on top of the one you are already executing in, so this might really be the only thing you can do. Perhaps it would be good to (1) clean up after yourself (uninstall the extension when you're done) and (2) pick a name with torch in it, so we don't accidentally clobber someone else's package.